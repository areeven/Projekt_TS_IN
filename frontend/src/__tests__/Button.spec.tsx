import renderer from 'react-test-renderer'
import styled from 'styled-components'

const Button = styled.button`
  color: red
`

test('renders button with styled-components', async () => {
    const button = renderer.create(<Button/>).toJSON()
    expect(button).toMatchSnapshot()
})
