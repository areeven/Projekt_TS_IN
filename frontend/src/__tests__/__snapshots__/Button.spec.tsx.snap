// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders button with styled-components 1`] = `
<button
  className="sc-bdvvtL gcYKbd"
/>
`;
